### PR TITLE
Remove k8s-master from dns names

### PR DESCRIPTION
**Pull request by brendandburns**
@colemickens

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import os.path
 import platform
 import random
+import socket
```
Comment (colemickens) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: Do you need this here?

**Comment by brendandburns** [comment]
Rebased and removed.  I may need to regenerate the unit test file...

**Comment by colemickens** [comment]
There's still a `-k8s-masters` in there, in `acs_create`. Looks like it's only used for the app's `identifier_uri` but you might still want to remove it for consistency's sake.

**Comment by brendandburns** [comment]
Removed and re-pushed.

**Comment by brendandburns** [comment]
@derekbekoe comment addressed, please re-check.

Thanks!
--brendan

**Pull request merged**